chemprop/nn/loss.py

```diff
@@ -81,7 +82,7 @@ def _calc_unreduced_loss(self, preds, targets, mask, weights, lt_mask, gt_mask)
         """Calculate a tensor of shape `b x t` containing the unreduced loss values."""
 
     def extra_repr(self) -> str:
-        return f"task_weights={self.task_weights.tolist()}"
+        return f"task_weights={self.task_weights}"
```

Comment:
```suggestion
        return f"task_weights={self.task_weights.tolist()}"
```
IMO a list is a clean `repr` than the raw tensor